### ISSUE TITLE
GraphQL-Java and DGS plugin for usage reporting

### ISSUE THREAD

**Issue by dotansimha** [graphql-hive/console#1430]
- GraphQL-Java
- DGS

**Comment by kliakos** [comment]
Is anybody currently involved with the development of a graphql-java solution for the operations reporting?

**Comment by n1ru4l** [comment]
@kliakos No, there is currently nobody involved. There is a specification on how to report usage data to the Hive registry: https://the-guild.dev/graphql/hive/docs/specs/usage-reports

**Comment by softprops** [comment]
We have one we have been using internal for graphql-java at Meetup for quite a while now. I'll have to write up a post about it to share what we are doing.

**Comment by kliakos** [comment]
> We have one we have been using internal for graphql-java at Meetup for quite a while now. I'll have to write up a post about it to share what we are doing.

That would be very helpful indeed.

We are also in the verge of writing an implementation and I wanted to see if anyone else has tried it successfully. And if yes, maybe we could discuss about open sourcing something for the public.
